### PR TITLE
NXP-21297: Add simple nightwatch.js IT ftest package

### PR DESCRIPTION
**Pull request by vpasquier**
@troger coucou!

### CODE REVIEW

File: ftest/nightwatchjs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>tomcat</id>
```
Comment (troger) [inline ftest/nightwatchjs/pom.xml]: You could remove the `tomcat`, `jboss` and `jbossear` profiles here. That's need to be done also on other ftest modules :(

Comment (jcarsique) [inline ftest/nightwatchjs/pom.xml]: :+1: the profiles are now inherited, no more need to duplicate them in child POM

File: ftest/nightwatchjs/tests/smoke_test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    'Sample Smoke Test' : function (browser) {
+        var nuxeo = browser.page.nuxeo();
+        nuxeo.navigate()
+        .waitForElementVisible('body', 1000)
+        .setValue('@username', 'Administrator')
+        .setValue('@password', 'Administrator')
+        .click('@submit')
+        .waitForElementVisible('@welcomemsg', 5000)
+        .assert.title('Nuxeo Platform - Domain')
+        .assert.containsText('@welcomemsg', 'Copyright © 2001-2016 Nuxeo and respective authors.');
```
Comment (jcarsique) [inline ftest/nightwatchjs/tests/smoke_test.js]: `Copyright © 2001-2016` ?

Comment (vpasquier) [inline ftest/nightwatchjs/tests/smoke_test.js]: What is the question?

File: ftest/nightwatchjs/pom.xml (hunk 1 above)
Comment (vpasquier) [inline ftest/nightwatchjs/pom.xml]: Ok

File: ftest/nightwatchjs/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Nuxeo Nightwatch Sample
+
```
Comment (vpasquier) [inline ftest/nightwatchjs/README.md]: There is no usage instruction, it's a really simple library to use.

I will try to find something without copy/pasting and without duplicating their website content/documentation...

**Comment by vpasquier** [comment]
Thank you guys for your reviews, here is my additional commit: https://github.com/nuxeo/nuxeo-marketplace-sample/pull/4/commits/35dae15290b7d47c2aa030333ad70df12164af4f

File: ftest/nightwatchjs/tests/smoke_test.js (hunk 2 above)
Comment (jcarsique) [inline ftest/nightwatchjs/tests/smoke_test.js]: Is the test checking that the page contains that text ?

Comment (vpasquier) [inline ftest/nightwatchjs/tests/smoke_test.js]: Yes totally. Maybe i could check for another thing (maybe more stable than a copyright) or just login, but i wanted to put one example of string checking and I guess in March we will have to check the new webui afterward. So I can comment it for instance if there is a concern about it.

**Comment by jcarsique** [comment]
:+1:

**Pull request merged**